### PR TITLE
{cae} [intel-2015b] OpenFOAM 2.4.0 with external dependenices (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by hajgato** [comment]
@boegel this is a simple version bump, needs to be tested, but in theory this should be straightforward to merge.

**Comment by boegel** [comment]
@hajgato: maybe #2023 first?

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@hajgato: please sync with develop now #2226 is merged in?

**Comment by hajgato** [comment]
@boegel done!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e21b0604a5c0c1c57fc8 for a full test report.

File: easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.4.0-intel-2015b-eb-deps-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+sources = [
+    SOURCE_TGZ,
+    'ThirdParty-%(version)s.tgz',
+]
+
+# fix for cyclic symlink issue, which may cause unpacking to fail
+# see also http://www.openfoam.org/mantisbt/view.php?id=1191
+# fixed in recent versions: https://github.com/OpenFOAM/OpenFOAM-2.3.x/commit/f7a485069c778495cc39b308580289f6c2d47163
+unpack_options = "--exclude=*tutorials/mesh/foamyHexMesh/mixerVessel/system/backgroundMeshDecomposition"
+unpack_options += " --exclude=*tutorials/mesh/foamyHexMesh/mixerVessel/system/cellShapeControlMesh"
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.4.0-intel-2015b-eb-deps-Python-2.7.10.eb]: this should be fixed in this version...

Comment (boegel) [inline easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.4.0-intel-2015b-eb-deps-Python-2.7.10.eb]: see the tag `OpenFOAM-2.4.0` at https://github.com/OpenFOAM/OpenFOAM-2.3.x/commit/f7a485069c778495cc39b308580289f6c2d47163

so, let's drop this here

**Comment by hajgato** [comment]
@boegel  Done.

**Comment by hajgato** [comment]
@boegel PR #2231

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bc42529e66394fb97ad4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**